src/SDKs/Azure.Base/data-plane/Azure.Base/ClientPipelineBuilderExtensions.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Net.Http;
+using Azure.Base.Http;
+using Azure.Base.Http.Pipeline;
+
+namespace Azure
+{
+    public static class ClientPipelineBuilderExtensions
```

Comment:
Why are these extensions and not instance methods on the builder? I think shared/common APIs should be instances, and then some services should have extensions with policies that are specific to the service.

Reply:
Can be both, personal preference. I would still keep AddPolicy methods off PipelineOptions but Append/Remove/etc can be moved to the instance.